src/util/rocksdb/mod.rs

```diff
@@ -305,27 +306,23 @@ pub fn delete_file_in_range(db: &DB, start_key: &[u8], end_key: &[u8]) -> Result
 
     for cf in db.cf_names() {
         let handle = try!(get_cf_handle(db, cf));
-
-        // Keys in CF_LOCK not have ts tail, at the same time DeleteFilesInRange treat
-        // input range as closed interval, but we don't want to delete the end_key, so
-        // we treat CF_LOCK especially. For the others column families: 1) the data set
-        // in these column families usually are very large, so we don't want to trigger
-        // seek for these column families; 2) keys in these column families have ts tail,
-        // so end_key never exists in these column families.
         if cf == CF_LOCK {
```

Comment:
I think we can ignore lock and raft and let outer clean up the region.

Reply:
This function only used in quickly cleanup data between regions gaps, there is no more
 other cleanup outer.

Reply:
So when we really clean up all data for the region? Will leave some stale data?

Reply:
Yes, will leave some stale data, but if we don't have a mechanism to roughly cleanup these stale data, there is a risk that the stale data may increase to very huge.